src/components/Reports/ReportEdit.tsx

```diff
@@ -196,13 +227,40 @@ export default function ReportEdit(): JSX.Element {
           additionalRightButtons={rightButtons}
         >
           {report &&
+            draftReport &&
             (showAnnual ? (
               <ReportFormAnnual editable={true} report={report} />
             ) : (
-              <ReportForm editable={true} report={report} onUpdateReport={updateReport} />
+              <ReportForm
+                editable={true}
+                draftReport={draftReport}
+                onUpdateReport={updateReport}
+                allSeedbanks={getAllSeedBanks(organization)
+                  .filter((f) => !!f)
```

Comment:
filter needed?

Reply:
The function `getAllSeedBanks` returns `(Facility | undefined)[]` so I thought it would be good to make sure to filter out any possible `undefined` values. I don't know if that ever occurs in practice, though.

Reply:
Got it, that util function should not return undefined.. seems like a bug. Should just be `Facility[]`
I thought we had fixed it..

Reply:
Oh, looks like I can simply change the return type signature and it's good to go. It looks correct for that return type already.